### PR TITLE
Mount all fix

### LINKED ISSUE #1032: runtime error on "blobfuse2 mount all"
### Which version of blobfuse was used?
2

### Which OS distribution and version are you using?
NAME="Ubuntu"
VERSION="20.04.5 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.5 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

running through WSL

### If relevant, please share your mount command.
blobfuse2 mount all /mnt/azstorage

### What was the issue encountered?
runtime error
```
panic: runtime error: index out of range [4] with length 4

goroutine 1 [running]:
github.com/Azure/azure-storage-fuse/v2/cmd.mountAllContainers(0xc00009e960, 0x2, 0x2, 0x0, 0x0, 0x7ffe7fbe3956, 0xe, 0x0, 0xe79982, 0x9)
        /mnt/vss/_work/1/s/azure-storage-fuse/cmd/mount_all.go:321 +0xf12
github.com/Azure/azure-storage-fuse/v2/cmd.processCommand(0x0, 0x0)
        /mnt/vss/_work/1/s/azure-storage-fuse/cmd/mount_all.go:180 +0x599
github.com/Azure/azure-storage-fuse/v2/cmd.glob..func9(0x149b920, 0xc00034b270, 0x1, 0x1, 0x0, 0x0)
        /mnt/vss/_work/1/s/azure-storage-fuse/cmd/mount_all.go:82 +0x9f
github.com/spf13/cobra.(*Command).execute(0x149b920, 0xc00034b250, 0x1, 0x1, 0x149b920, 0xc00034b250)
        /home/cloudtest/go/pkg/mod/github.com/gapra-msft/cobra@v1.4.1-0.20220411185530-5b83e8ba06dd/command.go:882 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0x149c100, 0x3, 0x3, 0xc00040a7c0)
        /home/cloudtest/go/pkg/mod/github.com/gapra-msft/cobra@v1.4.1-0.20220411185530-5b83e8ba06dd/command.go:1000 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/cloudtest/go/pkg/mod/github.com/gapra-msft/cobra@v1.4.1-0.20220411185530-5b83e8ba06dd/command.go:928
github.com/Azure/azure-storage-fuse/v2/cmd.Execute(0xffffffff, 0xc000042118)
        /mnt/vss/_work/1/s/azure-storage-fuse/cmd/root.go:262 +0x8d
main.main()
        /mnt/vss/_work/1/s/azure-storage-fuse/main.go:45 +0x25
```

### Have you found a mitigation/solution?
I looked at the code and it seems that the logic is expecting there to be 5 cli parms when there is only 4 in my command here...
https://github.com/Azure/azure-storage-fuse/blob/main/cmd/mount_all.go#L321

So I added an innocuous param (for me at least) `--disable-version-check=true`  and it seems to work now. 

I would expect an error message of some kind if it is missing a required parm rather than a runtime error

### PR DESCRIPTION
**Pull request by souravgupta-msft**
Fixes #1032

### CODE REVIEW

File: cmd/mount_test.go [hunk 1]
```diff
@@ -427,6 +427,26 @@ func (suite *mountTestSuite) TestFuseOptions() {
 	}
 }
 
+func (suite *mountTestSuite) TestUpdateCliParams() {
+	defer suite.cleanupTest()
+
+	cliParams := []string{"blobfuse2", "mount", "~/mntdir/", "--foreground=false"}
```
Comment (vibhansa-msft) [inline cmd/mount_test.go:433]: We shall  test for overwrite as well, like tmp-path is there in cliParams already then it shall not be added again or overwritten.

Comment (souravgupta-msft) [reply cmd/mount_test.go:433]: Already checking this below.
```
updateCliParams(&cliParams, "tmp-path", "tmpPath2")
updateCliParams(&cliParams, "container-name", "testCnt2")
suite.assert.Equal(len(cliParams), 6)
suite.assert.Equal(cliParams[4], "--tmp-path=tmpPath2")
suite.assert.Equal(cliParams[5], "--container-name=testCnt2")
```

File: cmd/mount_all.go [hunk 2]
```diff
@@ -317,8 +317,8 @@ func mountAllContainers(containerList []string, configFile string, mountPath str
 			cliParams[2] = "--config-file=" + contConfigFile
 		} else {
 			cliParams[2] = "--foreground=false"
-			cliParams[3] = "--container-name=" + container
-			cliParams[4] = "--tmp-path=" + filepath.Join(fileCachePath, container)
+			updateCliParams(&cliParams, "container-name", container)
```
[the remark this answers is not in the document]
Comment (souravgupta-msft) [reply cmd/mount_all.go:320]: Done

**Review by vibhansa-msft — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 - [#1002](https://github.com/Azure/azure-storage-fuse/issues/1002) In case version check fails to connect to public container, dump a log to check network and proxy settings.
 - [#1006](https://github.com/Azure/azure-storage-fuse/issues/1006) Remove user and group config from logrotate file.
 - [csi-driver #809](https://github.com/kubernetes-sigs/blob-csi-driver/issues/809) Fail to mount when uid/gid are provided on command line.
+- [#1032](https://github.com/Azure/azure-storage-fuse/issues/1032) CLI params updation fix in `mount all` command.
```
Comment (tasherif-msft) [inline CHANGELOG.md:7]: could you please explicitly mention here what the issue was - so for example "correctly parse and gracefully terminate on incorrect input parameters"

**Review by tasherif-msft — APPROVED** [review]

**Pull request merged**